tools/gdb/wt_debug_script_update.py

```diff
@@ -148,6 +148,29 @@ def dump_disk(leaf_page):
     page_bytes = gdb.selected_inferior().read_memory(int(dsk.address) + wt_page_header_size + wt_block_header_size, int(dsk['mem_size'])).tobytes()
     print("Dsk:\n" + str(page_bytes))
 
+def dump_leaf_page(leaf_page):
+    dbg('leaf', leaf_page)
+    dump_disk(leaf_page)
+    dump_modified(leaf_page)
+
+def dump_int_page(int_page):
+    dbg('Internal page', int_page)
+    pindex = int_page['u']['intl']['__index'].dereference()
+    dbg('pindex', pindex)
+    num_entries = int(pindex['entries'])
+    #dbg('rp-pre-index', rpindex['index'].dereference().dereference())
+    for i in range(0, num_entries):
+        if not pindex['index'][i].dereference()['page']:
+            continue
+        page = pindex['index'][i].dereference()['page'].dereference()
+        dbg('page', page)
+        page_type = page['type']
+        dbg('page type', page_type)
+        if page_type == 3 or page_type == 6:
```

Comment:
Could we add a short comment here to describe the page names corresponding with types 3 and 6?

Reply:
Added a comment. Please check and let me know.